src/root.component.tsx

```diff
@@ -58,6 +59,15 @@ export function Root(props: NavProps) {
     }
   });
 
+  React.useEffect(() => {
+    const abortController = new AbortController();
+    openmrsFetch("/ws/rest/v1/appui/session", {
```

Comment:
Isn't `sessionLocation` included in data returned from `getCurrentUser`?

Reply:
No. getCurrentUser only returns data specific to logged in user only. Location is included as part of session.

Reply:
Right, I wish the naming for `getCurrentUser` was better, but it does return the whole session object when parameter `includeAuthStatus: true` is provided. Try adding `console.log` to the `getCurrentUser({ includeAuthStatus: true }).subscribe(` callback. You should see `response.sessionLocation`.

Reply:
The typescript type of that function's return value might be incomplete, also.

Reply:
Sure. I will check that.

Reply:
It return something like this. It doesn't have location:
allowedLocales: (6) ["en", "en_GB", "es", "fr", "it", "pt"]
authenticated: true
locale: "en_GB"
sessionId: "7D7BD4DF9E550886B497D6CF31A408"
user:
display: "admin"
links: (2) [{…}, {…}]
person: {uuid: "24252571-dd5a-11e6-9d9c-0242ac1500", display: "Super User", links: Array(1)}
privileges: []
resourceVersion: "1.8"
retired: false
roles: (2) [{…}, {…}]
systemId: "admin"
userProperties: {loginAttempts: "0"}
username: ""
uuid: "45ce6c2e-dd5a-11e6-9d9c-0242ac1500"

Reply:
Huh! That is different from what I remember but it looks like you're right. Thanks for investigating.